src/extension/wps/src/test/java/au/org/emii/aggregator/NetcdfAggregatorTest.java

```diff
@@ -206,22 +206,32 @@ public void testLatLonProjectionSpatialSubsetNoData() throws IOException, Aggreg
         assertNetcdfFilesEqual(resourcePath("au/org/emii/aggregator/projection-point-outside-expected.nc"), outputFile);
     }
 
-    @Test
-    public void testAggregationOverrides() throws IOException, AggregationException {
+    private void testAggregationOverrides(String resourcePath) throws IOException, AggregationException {
```

Comment:
Minor thing - I'd normally put this below the methods that use it.  Normally you read java from top to bottom - higher level of abstraction at the top.